specification/storage/resource-manager/Microsoft.Storage/preview/2018-03-01-preview/blob.json

```diff
@@ -917,6 +964,54 @@
           }
         },
         "description": "The list of blob containers."
+      },
+      "LeaseContainerRequest": {
+        "properties": {
+          "action": {
+            "type": "string",
+            "enum": [
+              "Acquire",
+              "Renew",
+              "Change",
+              "Release",
+              "Break"
+            ],
+            "description": "Specifies the lease action. Can be one of the available actions."
+          },
+          "leaseId": {
+            "type": "string",
+            "description": "Identifies the lease. Can be specified in any valid GUID string format."
+          },
+          "breakPeriod": {
+            "type": "integer",
+            "description": "Optional. For a break action, proposed duration the lease should continue before it is broken, in seconds, between 0 and 60."
+          },
+          "leaseDuration": {
+            "type": "integer",
+            "description": "Required for acquire. Specifies the duration of the lease, in seconds, or negative one (-1) for a lease that never expires."
+          },
+          "proposedLeaseId": {
+            "type": "string",
+            "description": "Optional for acquire, required for change. Proposed lease ID, in a GUID string format."
+          }
+        },
+        "required": [
+          "action"
+        ],
+        "description": "Lease Container request schema."
+      },
+      "LeaseContainerResponse": {
+        "properties": {
+          "leaseId": {
+            "type": "string",
+            "description": "Returned unique lease ID that must be included with any request to delete the container, or to renew, change, or release the lease."
+          },
+          "leaseTimeSeconds": {
+            "type": "string",
```

Comment:
why this is a string, instead of number? also can we include this field in example?